frontend/ui/src/components/ColorSelector.tsx

```diff
@@ -12,22 +11,21 @@ import {
   cyTagsDialogColorSelectorPurple,
 } from 'handles';
 
+const mapStateToProps = (state: StateModel) => ({
+  tagConfig: state.data.config.tagConfig,
+});
+
+const connector = connect(mapStateToProps);
+
 type ColorSelectorProps = {
   handleUpdate: (event: React.ChangeEvent<HTMLInputElement>) => void;
   color: string;
   editing?: boolean;
   id?: string;
-};
-
-type ColorSelectorState = {
-  settings: Settings;
-};
+} & ConnectedProps<typeof connector>;
 
-const ColorSelector = ({handleUpdate, color, editing, id, settings}: ColorSelectorProps & ColorSelectorState) => {
-  const getColorValue = useCallback(
-    (color: string) => (settings && settings.colors[color].default) || '1578D4',
-    [settings]
-  );
+const ColorSelector = ({handleUpdate, color, editing, id, tagConfig}: ColorSelectorProps) => {
+  const getColorValue = useCallback((color: string) => tagConfig.colors[color]?.default ?? '1578D4', [tagConfig]);
```

Comment:
tagConfig should be optional as it might crash otherwise

Reply:
I defaulted it in the store so it can't be empty: https://github.com/airyhq/airy/pull/2838/files#diff-01a9a7cf1a6bdf2d5248c57d8f8b177ade69f004356f9f1bcb4812104a3aeef2R17